client/my-sites/backup/wpcom-backup-upsell.tsx

```diff
@@ -52,7 +52,7 @@ const BackupVPActiveBody: FunctionComponent = () => {
 	return (
 		<PromoCard
 			title={ preventWidows( translate( 'Your backups are powered by VaultPress' ) ) }
-			image={ <img src={ VaultPressLogo } width="155" alt="VaultPress logo" /> }
+			image={ { path: VaultPressLogo } }
```

Comment:
This change gives us the same DOM tree as we get for other PromoCard icons, making it easier to style with fewer rules.